<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>390</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/images/Settings.ico</normaloff>:/new/images/Settings.ico</iconset>
  </property>
  <widget class="QGroupBox" name="password_groupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Change Password</string>
   </property>
   <widget class="QLineEdit" name="oldpass_edit">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>33</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="newpass_edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>33</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="password_change_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>32</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="password_ok_button">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>32</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="oldpass_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>18</y>
      <width>71</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Old Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="newpass_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>18</y>
      <width>71</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>New Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="password_cancel_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>32</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="admin_groupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>521</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Admin Menu</string>
   </property>
   <widget class="QGroupBox" name="search_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>501</width>
      <height>65</height>
     </rect>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QRadioButton" name="order_radiobutton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>12</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Order No</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="quality_radiobutton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>12</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Quality</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="size_radiobutton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>12</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_lineedit">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>35</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="selection_disp_label">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>35</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Order No :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>34</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Will search database for the value given in the field"/>
     </property>
     <property name="whatsThis">
      <string extracomment="Will search database for the value given in the field">Will search database for the value given in the field</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_button">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>34</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Clears all fields"/>
     </property>
     <property name="whatsThis">
      <string extracomment="Clears all fields">Clears all fields</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="display_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>84</y>
      <width>501</width>
      <height>230</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>18</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>18</number>
    </attribute>
    <column>
     <property name="text">
      <string>Order ID</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weight</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quality</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tolerance</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tolerance %</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>search_lineedit</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>clear_button</tabstop>
  <tabstop>order_radiobutton</tabstop>
  <tabstop>quality_radiobutton</tabstop>
  <tabstop>size_radiobutton</tabstop>
  <tabstop>oldpass_edit</tabstop>
  <tabstop>password_ok_button</tabstop>
  <tabstop>newpass_edit</tabstop>
  <tabstop>password_change_button</tabstop>
  <tabstop>password_cancel_button</tabstop>
  <tabstop>display_table</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
